source/_integrations/aussie_broadband.markdown

```diff
@@ -0,0 +1,21 @@
+---
+title: Aussie Broadband
+description: Instructions on how to integrate Aussie Broadband within Home Assistant.
+ha_category:
+- Network
```

Comment:
The category & platforms seems a bit off? Should it also have a camera category?

Reply:
I wasn't sure what the possible "categories" were in this case, so just was guessing based on what I saw in other pages. Should this be `Network`, `Sensor`, and `Camera` (when `camera` platform is added in a subsequent PR)?

Reply:
I've added `Sensor` in the interim 😄 Will await your advice on this though

Reply:
Lets leave it at this.